autoload/neomake/makers/ft/php.vim

```diff
@@ -37,7 +37,8 @@ endfunction
 
 function! neomake#makers#ft#php#phpmd() abort
     return {
-        \ 'args': ['%:p', 'text', 'codesize,design,unusedcode,naming'],
+        \ 'args': ['%t', 'text', 'codesize,design,unusedcode,naming'],
+        \ 'append_file': 0,
```

Comment:
`phpmd` appears to require the file name to be in the position, correct @winged?